new_lineage_panel/src/App.tsx

```diff
@@ -374,6 +374,7 @@ function App() {
             setMoreTables({});
             vscode.postMessage({ command: "init" });
           }}
+          data-testid="reset-btn"
```

Comment:
can you use other locators like role as button with name attribute? As per best practices, better to use attributes visible to user rather than these abstracted ones
https://playwright.dev/docs/best-practices#use-locators
https://testing-library.com/docs/queries/bytestid/